src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala

```diff
@@ -545,17 +554,35 @@ abstract class OrganizeImports extends MultiStageRefactoring with TreeFactory wi
       else organizeImports(imports)
     }
 
+    private def organizeGroupedImports(stats: List[Tree])(importsOrganizer: List[Import] => List[Import])(transformer: Transformer): List[Tree] = {
+      stats.map { tree =>
+        List(tree)
+      }.foldLeft(List.empty[List[Tree]]) { (zero, elem) =>
```

Comment:
I would like to see some val declarations instead of this big method chains. That makes debugging easier. Alternatively you can fix the debugger to handle this syntax better.